shared/helpers.js

```diff
@@ -70,3 +71,10 @@ export function translate(format, bindings = {}) {
 
     return ret;
 }
+
+export let currency = {
+    find: c => currencies.find(curr => curr.code === c),
```

Comment:
Can you take the `find` function out of the export? It's used only privately here, so it could just be a local static function.